src/fides/api/ops/schemas/messaging/messaging.py

```diff
@@ -80,6 +80,7 @@ class AccessRequestCompleteBodyParams(BaseModel):
     """Body params required for privacy request completion access template"""
 
     download_links: List[str]
+    subject_request_download_time_in_days: Union[int, float]
```

Comment:
With this setup the message comes across as `Your data access has been completed and can be downloaded at your local fides_uploads folder. For security purposes, this secret link will expire in 5.0 days.`. If we were to change the type from `Union[int, float]` to `int` it would round down and drop the `.0` at the end. So `5.0` days would become `5` days and `5.9` days would also become `5` days.

I'm not sure if it would be better to show that it expires in `5.9` days or `5` days. Thoughts?

Reply:
I think rounding makes sense. I'll make that change 🙂